server/controllers/api.py

```diff
@@ -700,6 +704,29 @@ def can(action, resource, requester):
             return True
         return resource == requester
 
+class CourseEnrollment(Resource):
+    """ Information about all students in a course.
+    Authenticated. Permissions: >= User or admins
+    Used by: Export scripts.
+    """
+    model = models.Course
+    schema = CourseEnrollmentSchema()
+
+
+    @marshal_with(schema.get_fields)
+    def get(self, offering, user):
+        course = self.model.by_name(offering)
+        if course == None:
+            restful.abort(404)
+        if not self.model.can(course, user, 'staff'):
+            restful.abort(403)
+        data = {}
+        for role in VALID_ROLES:
```

Comment:
Nit/Doesn't need to be addressed - seems like a useful place for a defaultdict but I'm not 100% sure flask-restful will handle it (I think it will - but if this is only place we will use it then it's not worth importing)